docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -354,6 +354,38 @@ public class SomeTest {
 ----
 ====
 
+[[hr-named-datasource]]
+==== Named data sources
```

Comment:
See, this is where "simply" adding named datasources -- without named PUs -- complicates things.
Named datasources are mostly useful with named PUs, so in the case of Hibernate ORM extension, we only document named datasources in the context of named PUs, where it's easy to see the point: https://quarkus.io/guides/hibernate-orm#multiple-persistence-units
Here it's much less obvious why one would even bother using a named datasource.

Well, I suppose we can just rewrite the whole section in the same fashion when we add named PUs :)

Reply:
I agree, let's rewrite the section when multiple PUs are implemented as well